### ISSUE TITLE
signle quote not working

### ISSUE THREAD

**Issue by theowenyoung** [sunng87/handlebars-rust#470]
Test:

```rust
    #[test]
    fn test_template() {
        let mut handlebars = Handlebars::new();
        let data = json!({});
        handlebars.render_template("{{'test'}}", &data).unwrap();
    }
```

Throw, 
```bash
---- test::test_template stdout ----
thread 'test::test_template' panicked at 'called `Result::unwrap()` on an `Err` value: RenderError { desc: "Failed to parse template.", template_name: None, line_no: None, column_no: None, cause: Some(TemplateError { reason: InvalidSyntax, template_name: None, line_no: Some(1), column_no: Some(3), segment: Some("   0 | {{'test'}}\n     |---^------\n") }), unimplemented: false }', src/main.rs:155:57
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by sunng87** [comment]
Single quote is not supported for now. Is there any workaround for your scenario?

**Comment by theowenyoung** [comment]
Thanks for your replay!

I want use a helper: like `<html lang="{{first_non_empty lang 'en'}}">`

but I tried `<html lang="{{first_non_empty lang 'en'}}">`, `<html lang="{{first_non_empty lang [en]}}">`, `<html lang="{{first_non_empty lang \"en\"}}">`, neither of them can work.

Am I missed anything?

**Comment by theowenyoung** [comment]
Here is the first_non_empty helper code: https://github.com/davidB/handlebars_misc_helpers/blob/master/src/string_helpers.rs#L20

**Comment by sunng87** [comment]
@theowenyoung have you checked if `<html lang="{{first_non_empty lang "en"}}">` works for you?

**Comment by beltram** [comment]
Hi,

using single quotes actually works up to version `3.5.3`. It starts failing with [this commit](https://github.com/sunng87/handlebars-rust/commit/f9290a68b30fbc213662cea9e68133f1ee01d5cc) introduced in `3.5.4`.   

With my usage of handlebars, I must cope with a specification enforcing the usage of single quotes e.g. `{{now offset='3 days'}}`. It would be awesome to have single quote support back so that I could benefit from the improvements in the latest versions.  

@sunng87 WDYT should we re-open this one ?

**Comment by sunng87** [comment]
@beltram Thank you for investigation and we can reopen this.

**Comment by sunng87** [comment]
@beltram It should be fixed in #475. Could you please verify if the current master branch work you? I can create a release tomorrow if it's ok

**Comment by beltram** [comment]
@sunng87 indeed #475 works and fixes the issue. Thank you very much !

**Comment by sunng87** [comment]
@beltram the fix was released in 4.1.5
